src/mtype.d

```diff
@@ -7979,8 +7979,9 @@ public:
             e = e.semantic(sc);
             return e;
         }
-        auto de = new DotVarExp(e.loc, e, d);
```

Comment:
Personally I'd call ``de.toChars()`` from a debugger.  But I don't think that this is blocking anything.